app/assets/javascripts/admin/addon/components/schema-theme-setting/types/float.gjs

```diff
@@ -1,23 +1,13 @@
-import Component from "@glimmer/component";
-import { Input } from "@ember/component";
-import { on } from "@ember/modifier";
-import { action } from "@ember/object";
-import FieldInputDescription from "admin/components/schema-theme-setting/field-input-description";
+import SchemaThemeSettingNumberField from "admin/components/schema-theme-setting/number-field";
 
-export default class SchemaThemeSettingTypeFloat extends Component {
-  @action
-  onInput(event) {
-    this.args.onChange(parseFloat(event.currentTarget.value));
-  }
+export default class SchemaThemeSetitngTypeInteger extends SchemaThemeSettingNumberField {
+  step = 0.1;
 
-  <template>
-    <Input
-      @value={{@value}}
-      {{on "input" this.onInput}}
-      @type="number"
-      step="0.1"
-    />
+  parseValue(value) {
+    if (value === NaN) {
```

Comment:
I'm not sure if you know but 
<img width="101" alt="Screenshot 2024-03-21 at 3 14 36 PM" src="https://github.com/discourse/discourse/assets/1555215/6f227235-3cff-4b2b-a153-741d16bbb02e">

Reply:
O I didn't this is a good catch. I have to use `isNan` here then.